tc/core/halide2isl.cc

```diff
@@ -573,11 +574,12 @@ std::vector<Reduction> findReductions(const Stmt& s) {
     // For each init node, the names of its outer For nodes.
     std::map<std::string, std::vector<std::string>> initVars;
     std::map<std::string, Reduction> reductions;
+    std::map<std::string, Reduction> result;
```

Comment:
Why do you introduce a different field here?
The reductions are already collected in `reductions`.
Now, you may want to adjust the conditions under which reductions are
removed from consideration, but you should do it in a controlled way.